lua/gitlab/utils/init.lua

```diff
@@ -47,6 +47,33 @@ M.press_enter = function()
   vim.api.nvim_feedkeys(vim.api.nvim_replace_termcodes("<CR>", false, true, true), "n", false)
 end
 
+M.offset_to_seconds = function(offset)
```

Comment:
Hopefully this works in other timezones, it works perfectly for EST.

Basically we get the time from the MR, get the user's current timezone offset, convert that to seconds, and then add/subtract those seconds from the time of the MR (which is UTC). We then reconvert that numeric value back into a readable string. 

Parsing dates in Lua is difficult without any external dependencies!